src/components/Filters.js

```diff
@@ -103,10 +110,103 @@ export default class Filters {
   }
 
   reset() {
+    const rowNodes = document.querySelectorAll('[data-mt-filter-name]');
+    for (let i = 0; i < rowNodes.length; i++) {
+      rowNodes[i].parentNode.removeChild(rowNodes[i]);
+    }
     this.criteria = [];
-    this.container.innerHTML = '';
-    this.refresh();
-    this.maptable.map.reset();
+    this.maptable.render();
+  }
+
+  exportCriteria() {
```

Comment:
Almost no method have the signature documentation. I prefer to have verbose naming of variables and functions instead to save time.
If you'd like to do it while discovering the code, you're welcome to open a PR for this ;)